poutyne/framework/callbacks/color_formatting.py

```diff
@@ -200,3 +203,28 @@ def _get_formatted_metrics(self, metrics_str: str) -> str:
             formatted_metrics += self.text_color + name + ":" + self.metric_value_color + value
 
         return formatted_metrics
+
+    def _batch_update(self,
+                      update,
```

Comment:
I think that this argument argument is not necessary. This function should just return a new string and the += should be done by the caller.

Reply:
8bc604e